test/builder/graphql/timeline.js

```diff
@@ -32,7 +32,7 @@ export const CommitBuilder = createSpecBuilderClass('Commit', {
 
 export const CommitCommentBuilder = createSpecBuilderClass('CommitComment', {
   id: {default: nextID},
-  author: {linked: UserBuilder},
+  author: {linked: UserBuilder, default: null},
```

Comment:
You _could_ add `nullable: true` here too. All that really does is let you use the `.nullAuthor()` function on the generated builder, though, which is functionally equivalent to `.author(null)`.